.github/workflows/docker_release.yml

```diff
@@ -0,0 +1,69 @@
+name: One App Docker Release
+on:
+  push:
+    tags:
+      - '*'
+  repository_dispatch:
+    types: [one-app-docker-release]
+jobs:
+  one-app-docker-release:
+    name: Build and Deploy to Docker
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Set env
+        id: set_tag_version
+        run: |
+          if [ "${{ github.event.client_payload.tagversion }}" != "" ]; then
+          echo ::set-output name=tagversion::${{ github.event.client_payload.tagversion }}
+          else
+          echo ::set-output name=tagversion::$(echo $GITHUB_REF | cut -d / -f 3)
+          fi
+          echo $tagversion
+      - name: Docker login
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login --username ${{ secrets.DOCKER_USER }} --password-stdin
+      - name: Build production docker image
+        run: docker build -t prod .
+      - name: Extract and zip production statics
+        run: |
+          docker cp $(docker create prod):opt/one-app/build ./one-app-statics
+          zip -r one-app-static-assets one-app-statics/build
+      - name: Publish statics to NPM
+        run: npm publish ./one-app-statics --access public
```

Comment:
The version bump for the statics package happens during the Automatic PR that gets generated here: https://github.com/americanexpress/one-app/blob/ec63975f9e5db9b690258b5ea5988152a77e467d/package.json#L176 so no need to bump the `npm version` as it will take it from the `package.json`